lib/jormungandr/cardano-wallet-jormungandr.cabal

```diff
@@ -89,6 +89,7 @@ test-suite unit
     , cardano-crypto
     , cardano-wallet-jormungandr
     , containers
+    , directory
```

Comment:
we will need corresponding `nix` entry for this in `https://github.com/input-output-hk/cardano-wallet/blob/master/nix/.stack.nix/cardano-wallet-jormungandr.nix`

Reply:
adopted in https://github.com/input-output-hk/cardano-wallet/pull/633/commits/40766eeeef457a9fa4c9f0a641df6b9f02a9219e